### DISCUSSION TITLE
Welcome to Chat2DB Discussions!

### DISCUSSION THREAD

**Discussion by shanhexi** [OtterMind/Chat2DB#377] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by louis-gg** [comment]
希望加入类似cloudbeaver一样的功能，后台可以添加数据库
同时可以添加数据库给用户分配，用户登录后，直接列出有权限的数据库

**Comment by chenxinya** [comment]
希望可以 替换正常的数据库工具，但是现在基本的查询展示条数，变更影响条数，查询历史的功能等等还没有。

**Reply by shanhexi** [reply]
这些功能最新版本都有支持，可以更新一下最新版本

**Comment by luyh0114** [comment]
解释同一个SQL，第一次解释之后，把解释的窗口关闭了，没有办法找到关闭的窗口。然后进行第二次解释（同一个SQL），剩余的次数还是减少了，希望可以考虑一下缓存，同样的SQL解释直接快速返回，或者有办法找到关闭的解释窗口

**Comment by luyh0114** [comment]
感觉这是一项很有意义的工作，在使用了一下之后，查询效率确实有所提升。这是一个开源项目，如果缺少贡献者的话，希望可以有机会来参与这个开源项目后续的迭代开发，本人java后端开发

**Reply by shanhexi** [reply]
可以加群 联系我们哦，非常期待你的加入

**Comment by liuhd9** [comment]
看全表数据的时候没有筛选（快捷选字段查询）以及排序功能，这个很关键！！！

**Comment by liuhd9** [comment]
还有就是如果能整合PD 的功能进来 能表转 模型，模型 转表 就更酷了

**Comment by alexkeepfight** [comment]
求一个群，想一起commit代码

**Reply by shanhexi** [reply]
可以关注公众号Ch2DB,点击联系我们

**Comment by liubingmx** [comment]
![image](https://github.com/chat2db/Chat2DB/assets/20813546/596441d2-9be2-4c24-b573-43fee20a0995)
发现两个问题：
1. 询问当前mysql版本， 生成的sql不对，且"SQL查询:"注释没有加注释符，导致sql无法直接执行
2. 点击 "Format" 一直出 failed 提示，点一次出一次。。。。

**Comment by xiaoxinmin123** [comment]
有个特别想要的功能，就是在table表中control+F竟然不能过滤某个数据字段，要是有这个就可以完美平替其它工具了，希望引起大佬关注！！！！！
